### PR TITLE
Webhooks course learning video

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Add tip callout in the webhooks docs for the new on-demand webhooks learning course:

https://docs.getdbt.com/docs/deploy/webhooks

### CODE REVIEW

File: website/docs/docs/deploy/webhooks.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ You can create webhooks for these events from the [dbt Cloud web-based UI](#crea
 
 dbt Cloud retries sending each event five times. dbt Cloud keeps a log of each webhook delivery for 30 days. Every webhook has its own **Recent Deliveries** section, which lists whether a delivery was successful or failed at a glance. 
 
+:::tip Videos 
+If you're intersted in course learning with videos, check out the [on-demand Webhooks course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/webhooks.md]:
```suggestion
If you're intersted in course learning with videos, check out the [Webhooks on-demand course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
```

File: website/docs/docs/deploy/webhooks.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ You can create webhooks for these events from the [dbt Cloud web-based UI](#crea
 
 dbt Cloud retries sending each event five times. dbt Cloud keeps a log of each webhook delivery for 30 days. Every webhook has its own **Recent Deliveries** section, which lists whether a delivery was successful or failed at a glance. 
 
+:::tip Videos 
+If you're intersted in course learning with videos, check out the [Webhooks on-demand course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
+
+You can also check out the [dbt Fundamentals course](https://courses.getdbt.com/courses/fundamentals) that we offer for free.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/webhooks.md]:
```suggestion
You can also check out the free [dbt Fundamentals course](https://courses.getdbt.com/courses/fundamentals). 
```

File: website/docs/docs/deploy/webhooks.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ You can create webhooks for these events from the [dbt Cloud web-based UI](#crea
 
 dbt Cloud retries sending each event five times. dbt Cloud keeps a log of each webhook delivery for 30 days. Every webhook has its own **Recent Deliveries** section, which lists whether a delivery was successful or failed at a glance. 
 
+:::tip Videos 
+If you're intersted in course learning with videos, check out the [Webhooks on-demand course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/webhooks.md]: suggest simpler language and remove typo

```suggestion
If you prefer to learn through videos, check out the [Webhooks on-demand course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
```

**Review by mirnawong1 — CHANGES_REQUESTED** [review]
small suggestions @nghi-ly on this

File: website/docs/docs/deploy/webhooks.md (hunk 3 above)
Comment (nghi-ly) [inline website/docs/docs/deploy/webhooks.md]:
```suggestion
If you're interested in course learning with videos, check out the [Webhooks on-demand course](https://courses.getdbt.com/courses/webhooks) from dbt Labs.
```

Comment (nghi-ly) [reply website/docs/docs/deploy/webhooks.md]: good sugg! however, i do want to differentiate between any old video to "a structured course".

**Comment by nghi-ly** [comment]
thanks so much for the review, @mirnawong1 ! you rock!

**Review by mirnawong1 — APPROVED** [review]
🚢

**Comment by mirnawong1** [comment]
ready to merge for you @nghi-ly

**Pull request merged**